articles/imagesharp/pixelbuffers.md

```diff
@@ -67,7 +74,7 @@ A short summary of the limitations:
 
 **Non-conformant code:**
 ```C#
-Span<Rgba32> span = image.GetRowSpan(y);
+Span<Rgba32> span = imageFrame.PixelBuffer.DangerousGetRowSpan(y);
```

Comment:
I would delete the entire code sample. Today, we have analyzers to watch this + we need to avoid visibility of `imageFrame.PixelBuffer.DangerousGetRowSpan(y)`.

Reply:
Happy to do that 👍